assets/js/googlesitekit/modules/create-info-store.test.js

```diff
@@ -76,7 +83,7 @@ describe( 'createInfoStore store', () => {
 			// It uses `adminPage` when provided.
 			it( 'returns adminPage url when `adminPage` is provided', () => {
 				registry.dispatch( CORE_SITE ).receiveSiteInfo( { adminURL: 'http://example.com/wp-admin/' } );
-				const { STORE_NAME, ...store } = createInfoStore( MODULE_SLUG, { adminPage: 'test-admin-page' } );
+				const { STORE_NAME, ...store } = createInfoStore( MODULE_SLUG, { storeName: TEST_STORE_NAME, adminPage: 'test-admin-page' } );
```

Comment:
This line is fine going forward (in the context of other feedback above), I just wanted to highlight that the returned store name case was updated in some cases and not in others, so please make sure that `STORE_NAME` is destrucutred or otherwise referenced from the return in all instances.

Reply:
Based on the above comment, `createInfoStore` now returns `STORE_NAME` not `storeName` so I've reverted these to match.